test/calls/blockchain/transaction/get_transaction_test.exs

```diff
@@ -1,5 +1,102 @@
 defmodule ElectrumClient.Calls.Blockchain.Transaction.GetTransactionTest do
   use ExUnit.Case, async: true
+  alias ElectrumClient.Calls.Blockchain.Transaction.GetTransaction
 
-  doctest ElectrumClient.Calls.Blockchain.Transaction.GetTransaction
+  describe "translate" do
+    test "PSBT" do
```

Comment:
This thing is a mempool transaction that's not included in a block...

A PSBT is a completely different thing. Will merge this PR as the code is fine, will just work a bit on the test on a later date.